### DISCUSSION TITLE
The application crashes after entering the profile tab (iOS)

### DISCUSSION THREAD

**Discussion by MichalBurzynski89** [community/community#66968] [Mobile]
### Select Topic Area

Bug

### Body

Yesterday I noticed that I couldn't use the application, it kept closing after turning it on, I uninstalled the application and installed it again, I have the current iOS system and today I noticed that if I quickly change the tab to a tab other than profile, the application works, but I can't go to the profile, because it crashes every time, it's very irritating, what could be the problem?

**Comment by iskae** [comment]
Hi @MichalBurzynski89 👋 

Thanks so much for reaching out ❤️ We are already aware of the issue and the issue has already been fixed 🎉 Can you please check the latest version of the iOS app? 

Cheers!

**Reply by MichalBurzynski89** [reply]
I have the latest version, from yesterday, and my iOS version is 16.6.1 which seems up to date as well, but still it doesn't work, profile tab crashes the app.

**Comment by MichalBurzynski89** [comment]
It seems the problem doesn't exist anymore.
